### PR TITLE
Add helm charts to deploy to kubernetes

### PR DESCRIPTION
**Pull request by paulczar**
Signed-off-by: Paul Czarkowski <username.taken@gmail.com>

### CODE REVIEW

File: deployment/helm/event-sourcing-microservices-example/charts/edge-service [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/pczarkowski/development/scratch/event-sourcing-microservices-example/deployment/helm/edge-service
```
Comment (kbastani) [inline deployment/helm/event-sourcing-microservices-example/charts/edge-service]: @paulczar This doesn't look right. Can you x2 check?

File: deployment/helm/event-sourcing-microservices-example/charts/friend-service [hunk 2]
```diff
@@ -0,0 +1 @@
+/home/pczarkowski/development/scratch/event-sourcing-microservices-example/deployment/helm/friend-service
```
Comment (kbastani) [inline deployment/helm/event-sourcing-microservices-example/charts/friend-service]: @paulczar

File: deployment/helm/event-sourcing-microservices-example/charts/recommendation-service [hunk 3]
```diff
@@ -0,0 +1 @@
+/home/pczarkowski/development/scratch/event-sourcing-microservices-example/deployment/helm/recommendation-service
```
Comment (kbastani) [inline deployment/helm/event-sourcing-microservices-example/charts/recommendation-service]: @paulczar

File: deployment/helm/event-sourcing-microservices-example/charts/user-service [hunk 4]
```diff
@@ -0,0 +1 @@
+/home/pczarkowski/development/scratch/event-sourcing-microservices-example/deployment/helm/user-service
```
Comment (kbastani) [inline deployment/helm/event-sourcing-microservices-example/charts/user-service]: @paulczar

File: deployment/helm/friend-service/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "0.0.1-SNAPSHOT"
+description: A Helm chart for event-sourcing-microservices-example friend service
```
Comment (kbastani) [inline deployment/helm/friend-service/Chart.yaml]: Let's change the description to something a bit more generic. "Helm chart for the friend microservice in hyperskale/social-network-example"

File: deployment/helm/recommendation-service/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "0.0.2-SNAPSHOT"
+description: A Helm chart for event-sourcing-microservices-example recommendation service
```
Comment (kbastani) [inline deployment/helm/recommendation-service/Chart.yaml]: Let's change the description to something a bit more generic. "Helm chart for the recommendation microservice in hyperskale/social-network-example"

File: deployment/helm/recommendation-service/requirements.lock [hunk 7]
```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: neo4j
```
Comment (kbastani) [inline deployment/helm/recommendation-service/requirements.lock]: @paulczar @michael-simons How do we disable auth by default in the Neo4j helm chart?

File: deployment/helm/recommendation-service/values.yaml [hunk 8]
```diff
@@ -0,0 +1,52 @@
+# Default values for discovery-service.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: kbastani/recommendation-service
```
Comment (kbastani) [inline deployment/helm/recommendation-service/values.yaml]: We'll need to move this to hyperskale/recommendation-service

File: deployment/helm/user-service/Chart.yaml [hunk 9]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "0.0.2-SNAPSHOT"
+description: A Helm chart for event-sourcing-microservices-example user service
```
Comment (kbastani) [inline deployment/helm/user-service/Chart.yaml]: Let's change the description to something a bit more generic. "Helm chart for the user microservice in hyperskale/social-network-example"

File: deployment/helm/user-service/values.yaml [hunk 10]
```diff
@@ -0,0 +1,49 @@
+# Default values for discovery-service.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: kbastani/user-service
```
Comment (kbastani) [inline deployment/helm/user-service/values.yaml]: Change to hyperskale/user-service

**Comment by kbastani** [comment]
I've moved this pull request to a branch https://github.com/kbastani/event-sourcing-microservices-example/tree/paulczar-helm-charts.

File: deployment/helm/event-sourcing-microservices-example/charts/edge-service (hunk 1 above)
Comment (paulczar) [reply deployment/helm/event-sourcing-microservices-example/charts/edge-service]: oh dumb symlink bs! will fix.

File: deployment/helm/recommendation-service/requirements.lock (hunk 7 above)
Comment (paulczar) [reply deployment/helm/recommendation-service/requirements.lock]: if you check `deployment/helm/recommendation-service/values.yaml` you'll see the following set which disables auth.

```
neo4j.authEnabled: false
```

**Pull request merged**